addons/cookoff/XEH_postInit.sqf

```diff
@@ -62,6 +64,7 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 ["Car", "init", {
     params ["_vehicle"];
 
+    _vehicle setVariable [QGVAR(disableForAI), GVAR(disableForAI), true];
```

Comment:
Init event is globally executed, so why do you attempt to use setVariable public, which has global effects?
Aside from that, setVariable public is bugged in init events, it does not synchronize.

Reply:
Also by always setting this variable on every vehicle, using the global variable later as fallback setting (getVariable default value) becomes irrelevant. I would just delete this line completely as it also arbitrarily prevents the setting to take effect when changed in mission (but also only for vehicles that were created before the setting change).

I'd still keep it an object namespace variable that is by default undefined and falls back to the global setting, to allow mission makers (or anyone using class event handlers!) to enable or disable this for specific vehicles (vehicle types).